cmake/FindROOT.cmake

```diff
@@ -0,0 +1,36 @@
+# ------------------------------------------------------------------------
```

Comment:
We get rid off the FindROOT file recently, as ROOT provides its cmake Config file, so I do not want to re-introduce this file again. Please, see further that it is in fact not needed.

Reply:
There is no need for determining the ROOT cxx std flag as after finding ROOT we have
`include(${ROOT_USE_FILE})`